server/src/test/java/org/elasticsearch/index/search/geo/GeoUtilsTests.java

```diff
@@ -692,6 +694,34 @@ public void testParseGeoPointGeohashPositions() throws IOException {
             GeoUtils.EffectivePoint.BOTTOM_LEFT), new GeoPoint(42.890625, -71.3671875));
     }
 
+    public void testPointInBounds() {
```

Comment:
I think this goes away if we use lucene's `Rectangle#containsPoint` method since it''s unit tested by lucene?

Reply:
I do not see any explicit tests for this method in Lucene

https://github.com/apache/lucene-solr/search?q=containsPoint&unscoped_q=containsPoint

are there indirect tests for this?